html5lib/filters/whitespace.py

```diff
@@ -10,7 +10,7 @@
 
 
 class Filter(base.Filter):
-
+    """Collapses whitespace except in pre and textarea tags"""
```

Comment:
Elements, not tags. Also it's more than just pre and textarea; I think script is in rcdataElements too.